x-pack/plugins/fleet/public/applications/fleet/sections/debug/index.tsx

```diff
@@ -33,27 +42,70 @@ export const DebugPage: React.FunctionComponent = () => {
             Be aware that these debugging tools can be destructive in nature, and you should proceed with caution.`}
           />
 
-          <EuiSpacer size="xl" />
-          <AgentPolicyDebugger />
-          <EuiSpacer size="xl" />
-
-          <EuiHorizontalRule />
-
-          <EuiSpacer size="xl" />
-          <IntegrationDebugger />
-          <EuiSpacer size="xl" />
+          <EuiAccordion
```

Comment:
@kpollich added an accordion here, what do you think? I felt it might be useful to collapse the different sections, especially with the code blocks, the content can be long.

<img width="825" alt="image" src="https://user-images.githubusercontent.com/90178898/166679037-b055a006-10ae-41ca-88d4-897afd048dda.png">

Reply:
This is a good call. The page content was getting pretty noisy with all the various modules. I think this is a good solution for now!